<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_System">
     <property name="text">
      <string>Load System.csv</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_convertSystems">
     <property name="text">
      <string>Convert Systems</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_Station">
     <property name="text">
      <string>Load Station.csv</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_convertStations">
     <property name="text">
      <string>Convert Stations</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
